.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Release
+
+on:
+  release:
+    types: [prereleased, released]
+
+jobs:
+  release:
+    name: Release docker images and config
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: newrelic/infrastructure-k8s
+      S3_PATH: s3://nr-downloads-main/infrastructure_agent
+    steps:
+      - name: Generate docker image version from git tag
+        run: |
+          echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
+          DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
+          echo "$DOCKER_IMAGE_TAG" | grep -E '^[0-9.]*[0-9]$'
+          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
+
+      - if: ${{ github.event.release.prerelease }}
+        run: |
+          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG-pre" >> $GITHUB_ENV
+          echo "S3_PATH=$S3_PATH/test" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v2
+
+      - name: Build docker images
+        run: |
+          make compile
+          docker build -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG .
+          docker build -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG-unprivileged --build-arg 'MODE=unprivileged' .
+#          DOCKER_PRIVILEGED_SHA256=$(docker inspect --format='{{index (split (index .RepoDigests 0) "@sha256:") 1}}' $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG)
+#          DOCKER_UNPRIVILEGED_SHA256=$(docker inspect --format='{{index (split (index .RepoDigests 0) "@sha256:") 1}}' $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG-unprivileged)
+#          test "$DOCKER_PRIVILEGED_SHA256" != "$DOCKER_UNPRIVILEGED_SHA256"
```

Comment:
This check was present on Jenkins, although it cannot be directly ported here as locally generated images have an empty `.RepoDigests`. I can rewrite this to extract the SHA from the image ID if you think it is worth it, although I do not really see much value on it.

Reply:
We agreed this was not very useful and stripped it completely